js/webview/default.js

```diff
@@ -3,3 +3,7 @@
 var electron = require('electron')
 var ipc = electron.ipcRenderer
 var webFrame
+
+window.addEventListener('minUpdatedSettings', function () {
+	ipc.sendToHost('updatedSettings')
```

Comment:
We should check if the current page is a file:/// origin before sending an IPC message.